front/src/reducers/osrdconf/stdcmConf/selectors.ts

```diff
@@ -112,7 +112,7 @@ export const {
   getRetainedSimulationIndex,
   getWorkScheduleGroupId,
   getSearchDatetimeWindow,
-  getStdcmSelectedLayers,
+  getMapSettings,
```

Comment:
Hum we should not export `getMapSettings`, otherwise we can import it from any part of the code, but we should actually get the mapSettings through the hook useMapSettings